### DISCUSSION TITLE
Base Timezone

### DISCUSSION THREAD

**Discussion by lthon09** [deta/cloud-docs#322] [Help] [answered]
I'm wondering what the timezone for Base is, so that I can attach the expiration datetime object to an item. Would UTC be fine?

**Comment by lmangani** [comment, marked as answer]
Timestamps should be assumed UTC in general, but since `ExpireAt` also accepts a Date object you can specify a UTC relative TZ, ie: `new Date("2022-12-01T00:00:00.000+08:00");`

**Reply by lthon09** [reply]
got it :)
